src/lib/common/limits.h

```diff
@@ -94,6 +95,7 @@
 */
 #define  LOCAL_IP_V6  "::"
 #define  LOCAL_IP_V4  "0.0.0.0"
+#define  MAX_LEN_IP   64
```

Comment:
OK, looked up the format of IPv6:
```
2001:0db8:85a3:08d3:1319:8a2e:0370:7334
```

As a string, 8 chunks of 4 chars separated by 7 colons (and the ending 0): 8 * 5 == 40.
64 is more than enough